NYPLCardCreator/Classes/AlternativeAddressesViewController.swift

```diff
@@ -32,16 +32,31 @@ final class AlternativeAddressesViewController: TableViewController {
   
   override func viewDidLoad() {
     super.viewDidLoad()
+    
+    let headerView = UIView(frame: CGRect(x: 0, y: 0, width: self.view!.bounds.size.width, height: 80.0))
+    let headerLabel = UILabel()
+    headerView.addSubview(headerLabel)
+    
+    headerLabel.autoPinEdgesToSuperviewEdgesWithInsets(UIEdgeInsetsMake(2.0, 22.0, 2.0, 22.0))
+    headerLabel.numberOfLines = 2
```

Comment:
Again, fixing this at `2` makes things fragile given that the length of the text will change with different localizations.